src/App/App.js

```diff
@@ -9,34 +11,72 @@ class App extends Component{
     super()
 
     this.state = {
-      userData: {},
-      displayLoginForm : false
+      userData:  {
+        id: null,
+        name: '',
+        email: ''
+      },
     }
```

Comment:
I think the state should start with the information we are expecting from the server